simple_calculate_class.py

```diff
@@ -1,15 +1,24 @@
+import numpy as np
+
+
 class Calculate:
-    def add(self, a: int, b: int):
-        if not (isinstance(a, int) and isinstance(b, int)):
-            raise TypeError('Input must be int.')
-        return a + b
+    def add(self, *nums):
+        for num in nums:
+            if not isinstance(num, int):
+                raise TypeError("Inputs must be integers")
+                return
+
+        return np.sum(nums)
```

Comment:
what is actually returned type of `add()` ?
- add(2)
- add()
is it better the same type with `sub()` ?

Reply:
the returned type was  'numpy.int64'. 
I have revised it backed to 'int'